.travis.yml

```diff
@@ -28,12 +28,7 @@ install:
   # TODO: IPython 2 env as well
   - conda install pip ipython-notebook
```

Comment:
this is about my first project that doesn't depend on scientific packages, so I might not need conda here, and pip might be good enough for all dependencies... I'm not sure about what the best practices are here: what do you think?